### PR TITLE
README.md: Windows support; binary name changes; vsettings script; etc.

### LINKED ISSUE #354: TODO: Update README with simpler instructions for installing dependencies, using script/bootstrap

### PR DESCRIPTION
**Pull request by stephengtuggy**
Thank you for submitting a pull request and becoming a contributor to the Vega Strike Core Engine.

Please answer the following:

Code Changes:
- [ ] Have the PR Validation Tests been run? See https://github.com/vegastrike/Vega-Strike-Engine-Source/wiki/Pull-Request-Validation
- [X] This is a documentation change only

Issues:
- Please list any related issues
- Closes #354 
- Partially addresses #123 
- Closes https://github.com/vegastrike/Assets-Masters/issues/23

Purpose:
- What is this pull request trying to do? Port #503 to master
- What release is this for? 0.9.x/1.0.x
- Is there a project or milestone we should apply this to? 0.9.x/1.0.x

### CODE REVIEW

**Review by BenjamenMeyer — CHANGES_REQUESTED** [review]
Couple minor changes; overall looking good.

**Review by Loki1950 — APPROVED** [review]
With Ben's changes

File: README.md [hunk 1]
```diff
@@ -85,26 +85,33 @@ OPTIONS
...
+Also note that when you install vsUtCS, it comes with a script called `vsettings` that automatically runs `vegasettings` with the correct data directory, and another script called `vs` that automatically runs `vegastrike-engine` with the correct data directory. These scripts are the recommended way to run the game Vega Strike: Upon the Coldest Sea. You should see shortcuts to them on your desktop, Start Menu, or similar.
 
 If you encounter any issues while playing, please create an issue with the Vega Strike development team by [posting a new issue](https://github.com/vegastrike/Vega-Strike-Engine-Source/issues).
 
 REQUIRED FILES
 --------------
 
 ```bash
-  /usr/local/bin/vegastrike
+  /usr/local/bin/vegastrike-engine
```
[the remark this answers is not in the document]
Comment (stephengtuggy) [reply README.md]: Done.

File: README.md [hunk 2]
```diff
@@ -58,12 +58,12 @@ How to Run
 Either install Vega Strike from the binary installer for your platform, if available, or follow the instructions for compiling from source. (`Compiling Vegastrike`, below.)
 
 - First change settings with `bin/vegasettings`
-- Then run `bin/vegastrike` or double-click on the executable file in the bin directory.
+- Then run `bin/vegastrike-engine` or double-click on the executable file in the bin directory.
```
[the remark this answers is not in the document]
Comment (stephengtuggy) [reply README.md]: Fixed

**Review by BenjamenMeyer — APPROVED** [review]

**Pull request merged**